configurations/config.ini.example

```diff
@@ -0,0 +1,7 @@
+[PSQL]
+ENGINE = django.db.backends.postgresql
+NAME =
```

Comment:
I think it's self-evident because it's the default key. Perhaps it is better to change the [PSQL] into something like a [PostgreSQL_DB] or a [PSQL_DB]. And later there will be a [Mongo_DB] too.
But anyway it doesn't matter. If everyone agrees I'll rename it.